### PR TITLE
Create peter-toma.md

### PR DESCRIPTION
**Pull request by ovesyan**
This article appears to be more about the product rather than the developer. There is very little about him.

### CODE REVIEW

**Comment by bittlingmayer** [comment]
You can find a bit more at:
https://ru.wikipedia.org/wiki/SYSTRAN and https://fr.wikipedia.org/wiki/Systran

For example, that he was at Caltech.

https://fr.wikipedia.org/wiki/Peter_Toma

File: history/peter-toma.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+description: Programmer who developed the oldest commercial machine translation program
```
Comment (bittlingmayer) [inline history/peter-toma.md:2]: > Developer of the first machine translation product

File: history/peter-toma.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+---
+# Peter Toma
+Peter Toma was a computer programmer who designed Systran, the oldest commercial machine translation program. 
+
+He initially developed the program for the United States Air Force. 
+It was intended to translate large-scale scientific documents from Russian into English during the Cold War. 
+
+It was later acquired by the European Commission for English-to-French direction.
```
Comment (bittlingmayer) [inline history/peter-toma.md:10]: > The European Commission acquired it for translating from English to French.

Did the acquire the whole thing?  Or did they just become one customer among others?

File: history/peter-toma.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+description: Programmer who developed the oldest commercial machine translation program
+---
+# Peter Toma
+Peter Toma was a computer programmer who designed Systran, the oldest commercial machine translation program. 
+
+He initially developed the program for the United States Air Force. 
+It was intended to translate large-scale scientific documents from Russian into English during the Cold War. 
+
+It was later acquired by the European Commission for English-to-French direction. 
+The language pairs got expanded when the program was purchased and popularized by a French company.
```
Comment (bittlingmayer) [inline history/peter-toma.md:11]: Let's just make an article about Systran, and this kind of thing can go there.

We can say that he was at Caltech.  Was he at Georgetown too?

File: history/peter-toma.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+description: Developer of the first machine translation product
+---
+# Peter Toma
+Peter Toma was a computer scientist and linguistics researcher. 
+
+In 1956, Toma developed software for machine translation at the California Institute of Technology. 
+The system translated Russian texts into English. Later, he worked at Georgetown University where the system 
+Serna was developed for the same translation direction. 
+
+In 1962, he founded a company which created the systems Autotran and Technotran.
```
Comment (bittlingmayer) [inline history/peter-toma.md:11]: What was the name of the company?

File: history/peter-toma.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+description: Developer of the first machine translation product
+---
+# Peter Toma
+Peter Toma was a computer scientist and linguistics researcher. 
+
+In 1956, Toma developed software for machine translation at the California Institute of Technology. 
+The system translated Russian texts into English. Later, he worked at Georgetown University where the system 
+Serna was developed for the same translation direction.
```
Comment (bittlingmayer) [inline history/peter-toma.md:9]: >worked at Georgetown University on the development of the Serna system, which was also for translating Russian to English.

i.e. avoid passive (but also avoid saying "he developed" as if it was only him)

File: history/peter-toma.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+---
+# Peter Toma
+Peter Toma was a computer scientist and linguistics researcher. 
+
+In 1956, Toma developed software for machine translation at the California Institute of Technology. 
+The system translated Russian texts into English. Later, he worked at Georgetown University where the system 
+Serna was developed for the same translation direction. 
+
+In 1962, he founded a company which created the systems Autotran and Technotran. 
+They were specialized in Russian-to-English translation in the fields of atomic energy and medicine.
```
Comment (bittlingmayer) [inline history/peter-toma.md:12]: They were for translating Russian to English and specialized for translating content about atomic energy and medicine.

File: history/peter-toma.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+Peter Toma was a computer scientist and linguistics researcher. 
+
+In 1956, Toma developed software for machine translation at the California Institute of Technology. 
+The system translated Russian texts into English. Later, he worked at Georgetown University where the system 
+Serna was developed for the same translation direction. 
+
+In 1962, he founded a company which created the systems Autotran and Technotran. 
+They were specialized in Russian-to-English translation in the fields of atomic energy and medicine.  
+
+In the mid-1960s, with the support of the German Research Foundation Toma programmed the [Systran](/companies/systran.md) prototype.
```
Comment (bittlingmayer) [inline history/peter-toma.md:14]: > In the mid-1960s, Toma programmed the [Systran](/business/companies.md#systran) prototype, with the support of the German Research Foundation.

Or at least put a comma after *Foundation*...  Otherwise it becomes hard to parse.

Also the link should be to `/business/companies.md#systran` - because there is not such page

File: history/peter-toma.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
+---
+description: Developer of the first machine translation product
+---
+# Peter Toma
+Peter Toma was a computer scientist and linguistics researcher. 
+
```
Comment (bittlingmayer) [inline history/peter-toma.md:6]: Do we know where he studied?  He was born in Hungary, I have a feeling that he left in 1940, when he was 16.

File: history/peter-toma.md (hunk 4 above)
Comment (ovesyan) [reply history/peter-toma.md:11]: The article says "his own company". It doesn't give a name, and I couldn't find it.

File: history/peter-toma.md (hunk 8 above)
Comment (ovesyan) [reply history/peter-toma.md:6]: There is nothing about his education, unfortunately.

File: history/peter-toma.md (hunk 4 above)
Comment (bittlingmayer) [reply history/peter-toma.md:11]: Maybe we should create Issues for these types of things?
